app/controllers/refinery/authentication/devise/admin/users_controller.rb

```diff
@@ -47,7 +47,7 @@ def update
 
             store_user_memento
 
-            @user.roles = @selected_role_names.map { |r| Refinery::Authentication::Devise::Role[r.downcase] }
+            @user.roles = @selected_role_names.map { |r| Refinery::Authentication::Devise::Role[r.underscore] }
```

Comment:
@dreamsoftech down on lines 66 and 122 it's still using `Refinery::Authentication::Devise::Role[r.downcase]` and `@selected_role_names.map(&:downcase)`

Reply:
@parndt Good Catch!